src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Color getRandomColor() {
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/12864128/91b9e60a-0882-4064-806d-4b4266e0ee36)

Reply:
Just static? (it was Int.Idea. solution and i had aggree)
i can switch type  to Color?